### ISSUE TITLE
Unable to build GraalVM micronaut app with OkHttp dependency

### ISSUE THREAD

**Issue by zsiegel** [oracle/graal#2765]
**Describe the issue**
Hello. I have been testing how to build my micronaut api with graalvm via Docker. Happy to provide anymore output and a sample project would just need some time to cleanup/remove some sensitive bits.

Currently during build time I experience an issue compiling due to `java.lang.NoClassDefFoundError: okhttp3/logging/HttpLoggingInterceptor$Logger$DefaultImpls`

**Steps to reproduce the issue**
1. Compile the app via `docker build -f Dockerfile.graalvm . -t alpha-graalvm`

**Describe GraalVM and your environment:**
 - GraalVM version **Docker CE 20.1.0-java11**
 - JDK major version: **11**
 - OS: **Docker on macOS Catalina**
 - Architecture: **AMD64**

**More details**

Dockerfile
```
FROM gradle:6.6-jdk11 as gradle
COPY --chown=gradle . /home/app
WORKDIR /home/app
RUN gradle assemble

# Stage 2: Build the native image
FROM oracle/graalvm-ce:20.1.0-java11 as graalvm
COPY --from=gradle /home/app/build/libs/alpha-api-0.1-all.jar /home/app/
WORKDIR /home/app
RUN gu install native-image
RUN native-image --native-image-info --verbose --no-server --initialize-at-build-time=net.logstash.logback,org.conscrypt.OpenSSLProvider,org.conscrypt.Conscrypt -cp alpha-api-0.1-all.jar

FROM frolvlad/alpine-glibc
EXPOSE 8080
COPY --from=graalvm /home/app/alpha-graalvm .
ENTRYPOINT ["./alpha-graalvm"]
```

Docker log output
```
docker build -f Dockerfile.graalvm . -t alpha-graalvm
Sending build context to Docker daemon  153.6kB
Step 1/13 : FROM gradle:6.6-jdk11 as gradle
 ---> d94ba6caf973
Step 2/13 : COPY --chown=gradle . /home/app
 ---> 5b7220499eff
Step 3/13 : WORKDIR /home/app
 ---> Running in ff9afab4f439
Removing intermediate container ff9afab4f439
 ---> beb345830b09
Step 4/13 : RUN gradle assemble
 ---> Running in 92c43439af90

Welcome to Gradle 6.6!

Here are the highlights of this release:
 - Experimental build configuration caching
 - Built-in conventions for handling credentials
 - Java compilation supports --release flag

For more details see https://docs.gradle.org/6.6/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
> Task :kaptGenerateStubsKotlin
> Task :processResources

> Task :kaptKotlin
Note: Creating bean classes for 29 type elements[WARN] Incremental annotation processing requested, but support is disabled because the following processors are not incremental: io.micronaut.annotation.processing.TypeElementVisitorProcessor (DYNAMIC), io.micronaut.annotation.processing.AggregatingTypeElementVisitorProcessor (DYNAMIC), io.micronaut.annotation.processing.PackageConfigurationInjectProcessor (DYNAMIC), io.micronaut.annotation.processing.BeanDefinitionInjectProcessor (DYNAMIC), io.micronaut.annotation.processing.ServiceDescriptionProcessor (DYNAMIC).

> Task :compileKotlin
> Task :compileJava NO-SOURCE
> Task :classes
> Task :inspectClassesForKotlinIC
> Task :jar
> Task :startScripts
> Task :distTar
> Task :distZip
> Task :shadowJar
> Task :startShadowScripts
> Task :shadowDistTar
> Task :shadowDistZip
> Task :assemble

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1m 11s
13 actionable tasks: 13 executed
Removing intermediate container 92c43439af90
 ---> 4a3a1806c82b
Step 5/13 : FROM oracle/graalvm-ce:20.1.0-java11 as graalvm
 ---> 2adef7aab8f9
Step 6/13 : COPY --from=gradle /home/app/build/libs/alpha-api-0.1-all.jar /home/app/
 ---> 35781e6d1cbe
Step 7/13 : WORKDIR /home/app
 ---> Running in 5897e9d3be76
Removing intermediate container 5897e9d3be76
 ---> 9424a4a1b52a
Step 8/13 : RUN gu install native-image
 ---> Running in 1b3285ac5a96
Downloading: Component catalog from www.graalvm.org
Processing Component: Native Image
Downloading: Component native-image: Native Image  from github.com
Installing new component: Native Image (org.graalvm.native-image, version 20.1.0)
Refreshed alternative links in /usr/bin/
Removing intermediate container 1b3285ac5a96
 ---> 1906d8d40ea2
Step 9/13 : RUN native-image --native-image-info --verbose --no-server --initialize-at-build-time=net.logstash.logback,org.conscrypt.OpenSSLProvider,org.conscrypt.Conscrypt -cp alpha-api-0.1-all.jar
 ---> Running in 7b1cf7951487
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut.core/cache-caffeiene-graal/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/common/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/buffer/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/transport/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/handler/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/codec-http/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.netty/codec-http2/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut.http/mime-types/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/inject/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/micronaut-http/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/netty/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/micronaut-http-netty/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/runtime-graal/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/micronaut-runtime/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/micronaut-http-server-netty/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/com.centerboard.market/complete-application/native-image.properties
Apply jar:file:///home/app/alpha-api-0.1-all.jar!/META-INF/native-image/io.micronaut/micronaut-http/resource-config.json
Executing [
/opt/graalvm-ce-java11-20.1.0/bin/java \
-XX:+UseParallelGC \
-XX:+UnlockExperimentalVMOptions \
-XX:+EnableJVMCI \
-Dtruffle.TrustAllTruffleRuntimeProviders=true \
-Dtruffle.TruffleRuntime=com.oracle.truffle.api.impl.DefaultTruffleRuntime \
-Dgraalvm.ForcePolyglotInvalid=true \
-Dgraalvm.locatorDisabled=true \
-Dsubstratevm.IgnoreGraalVersionCheck=true \
-Djava.lang.invoke.stringConcat=BC_SB \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.runtime=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.code=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.aarch64=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.amd64=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.meta=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.hotspot=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.services=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.common=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.code.site=ALL-UNNAMED \
--add-exports \
jdk.internal.vm.ci/jdk.vm.ci.code.stack=ALL-UNNAMED \
--add-opens \
jdk.internal.vm.compiler/org.graalvm.compiler.debug=ALL-UNNAMED \
--add-opens \
jdk.internal.vm.compiler/org.graalvm.compiler.nodes=ALL-UNNAMED \
--add-opens \
jdk.unsupported/sun.reflect=ALL-UNNAMED \
--add-opens \
java.base/jdk.internal.module=ALL-UNNAMED \
--add-opens \
java.base/jdk.internal.ref=ALL-UNNAMED \
--add-opens \
java.base/jdk.internal.reflect=ALL-UNNAMED \
--add-opens \
java.base/java.io=ALL-UNNAMED \
--add-opens \
java.base/java.lang=ALL-UNNAMED \
--add-opens \
java.base/java.lang.reflect=ALL-UNNAMED \
--add-opens \
java.base/java.lang.invoke=ALL-UNNAMED \
--add-opens \
java.base/java.lang.ref=ALL-UNNAMED \
--add-opens \
java.base/java.net=ALL-UNNAMED \
--add-opens \
java.base/java.nio=ALL-UNNAMED \
--add-opens \
java.base/java.nio.file=ALL-UNNAMED \
--add-opens \
java.base/java.security=ALL-UNNAMED \
--add-opens \
java.base/javax.crypto=ALL-UNNAMED \
--add-opens \
java.base/java.util=ALL-UNNAMED \
--add-opens \
java.base/java.util.concurrent.atomic=ALL-UNNAMED \
--add-opens \
java.base/sun.security.x509=ALL-UNNAMED \
--add-opens \
java.base/jdk.internal.logger=ALL-UNNAMED \
--add-opens \
org.graalvm.sdk/org.graalvm.nativeimage.impl=ALL-UNNAMED \
--add-opens \
org.graalvm.sdk/org.graalvm.polyglot=ALL-UNNAMED \
--add-opens \
org.graalvm.truffle/com.oracle.truffle.polyglot=ALL-UNNAMED \
--add-opens \
org.graalvm.truffle/com.oracle.truffle.api.impl=ALL-UNNAMED \
-XX:+UseJVMCINativeLibrary \
-Xss10m \
-Xms1g \
-Xmx6681601632 \
-Duser.country=US \
-Duser.language=en \
-Djava.awt.headless=true \
-Dorg.graalvm.version=20.1.0 \
-Dorg.graalvm.config= \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Xshare:off \
--module-path \
/opt/graalvm-ce-java11-20.1.0/lib/truffle/truffle-api.jar \
-javaagent:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/svm.jar \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-cp \
/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/llvm-wrapper-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/objectfile.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/svm-llvm.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/javacpp-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/pointsto.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/llvm-platform-specific-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/svm.jar \
'com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus' \
-watchpid \
1 \
-imagecp \
/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/llvm-wrapper-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/objectfile.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/svm-llvm.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/javacpp-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/pointsto.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/llvm-platform-specific-shadowed.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/builder/svm.jar:/opt/graalvm-ce-java11-20.1.0/lib/svm/library-support.jar:/home/app/alpha-api-0.1-all.jar \
-H:Path=/home/app \
-H:+DumpTargetInfo \
-H:ClassInitialization=net.logstash.logback:build_time,org.conscrypt.OpenSSLProvider:build_time,org.conscrypt.Conscrypt:build_time \
-H:ReflectionConfigurationResources=META-INF/native-image/io.micronaut.core/cache-caffeiene-graal/reflection-config.json \
-H:ClassInitialization=io.netty.util.AbstractReferenceCounted:run_time,io.netty.util.concurrent.GlobalEventExecutor:run_time,io.netty.util.concurrent.ImmediateEventExecutor:run_time,io.netty.util.concurrent.ScheduledFutureTask:run_time,io.netty.util.internal.ThreadLocalRandom:run_time \
-H:ClassInitialization=io.netty.buffer.PooledByteBufAllocator:run_time,io.netty.buffer.ByteBufAllocator:run_time,io.netty.buffer.ByteBufUtil:run_time,io.netty.buffer.AbstractReferenceCountedByteBuf:run_time \
-H:ReflectionConfigurationResources=META-INF/native-image/io.netty/transport/reflection-config.json \
-H:ClassInitialization=io.netty.handler.ssl.util.ThreadLocalInsecureRandom:run_time \
-H:ClassInitialization=io.netty.handler.codec.http.HttpObjectEncoder:run_time,io.netty.handler.codec.http.websocketx.WebSocket00FrameEncoder:run_time,io.netty.handler.codec.http.websocketx.extensions.compression.DeflateDecoder:run_time \
-H:ClassInitialization=io.netty:build_time \
-H:ClassInitialization=io.netty.handler.codec.http2.Http2CodecUtil:run_time,io.netty.handler.codec.http2.Http2ClientUpgradeCodec:run_time,io.netty.handler.codec.http2.Http2ConnectionHandler:run_time,io.netty.handler.codec.http2.DefaultHttp2FrameWriter:run_time \
-H:IncludeResources=META-INF/http/mime.types \
-H:ClassInitialization=io.micronaut.http.bind.binders.ContinuationArgumentBinder:run_time \
-H:+AllowIncompleteClasspath \
-H:EnableURLProtocols=http,https \
-H:ReflectionConfigurationResources=META-INF/native-image/io.micronaut/micronaut-http/reflection-config.json \
'-H:ClassInitialization=com.sun.jndi.dns.DnsClient:run_time,io.netty.handler.ssl.ConscryptAlpnSslEngine:run_time,io.netty.handler.ssl.JettyNpnSslEngine:run_time,io.netty.handler.ssl.ReferenceCountedOpenSslEngine:run_time,io.netty.handler.ssl.JdkNpnApplicationProtocolNegotiator:run_time,io.netty.handler.ssl.ReferenceCountedOpenSslServerContext:run_time,io.netty.handler.ssl.ReferenceCountedOpenSslClientContext:run_time,io.netty.handler.ssl.util.BouncyCastleSelfSignedCertGenerator:run_time,io.netty.handler.ssl.ReferenceCountedOpenSslContext:run_time,io.micronaut.buffer.netty.NettyByteBufferFactory:run_time,io.netty.handler.ssl.JettyAlpnSslEngine$ClientEngine:run_time,io.netty.handler.ssl.JettyAlpnSslEngine$ServerEngine:run_time,io.netty.handler.codec.http2.Http2CodecUtil:run_time,io.netty.handler.codec.http2.CleartextHttp2ServerUpgradeHandler:run_time,io.netty.handler.codec.http2.Http2ServerUpgradeCodec:run_time,io.micronaut.http.netty.channel.converters.EpollChannelOptionFactory:run_time,io.micronaut.http.netty.channel.converters.KQueueChannelOptionFactory:run_time,io.micronaut.http.bind.binders.ContinuationArgumentBinder$Companion:run_time,io.micronaut.http.bind.binders.ContinuationArgumentBinder:run_time' \
-H:ReflectionConfigurationResources=META-INF/native-image/io.micronaut/micronaut-http-netty/reflection-config.json \
-H:ClassInitialization=io.micronaut.reactive.reactor.ReactorInstrumentation:run_time \
-H:ClassInitialization=ch.qos.logback:build_time,com.fasterxml.jackson:build_time,io.micronaut:build_time,io.reactivex:build_time,org.reactivestreams:build_time,org.slf4j:build_time,org.yaml.snakeyaml:build_time,javax:build_time \
-H:ReflectionConfigurationResources=META-INF/native-image/io.micronaut/micronaut-runtime/reflection-config.json \
-H:ReflectionConfigurationResources=META-INF/native-image/io.micronaut/micronaut-http-server-netty/reflection-config.json \
-H:Name=alpha-graalvm \
-H:Class=com.centerboard.market.Application \
-H:ResourceConfigurationResources=META-INF/native-image/io.micronaut/micronaut-http/resource-config.json \
-H:CLibraryPath=/opt/graalvm-ce-java11-20.1.0/lib/svm/clibraries/linux-amd64 \

]
[alpha-graalvm:24]    classlist:   8,811.30 ms,  1.19 GB
[alpha-graalvm:24]        (cap):     517.34 ms,  1.19 GB
[alpha-graalvm:24]        setup:   1,897.33 ms,  1.19 GB
# Building image for target platform: org.graalvm.nativeimage.Platform$LINUX_AMD64
# Using native toolchain:
#   Name: GNU project C and C++ compiler (gcc)
#   Vendor: redhat
#   Version: 4.8.5
#   Target architecture: x86_64
#   Path: /usr/bin/gcc
# Using CLibrary: com.oracle.svm.core.c.libc.GLibc
WARNING GR-10238: VarHandle for static field is currently not fully supported. Static field private static volatile java.lang.System$Logger jdk.internal.event.EventHelper.securityLogger is not properly marked for Unsafe access!
[alpha-graalvm:24]     analysis:  41,559.79 ms,  3.64 GB
```

StackTrace below
```
Fatal error:com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing okhttp3.logging.HttpLoggingInterceptor$Logger.<clinit>() 
Parsing context: <no parsing context available> 

	at com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:138)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.doParse(MethodTypeFlow.java:336)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureParsed(MethodTypeFlow.java:311)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.addContext(MethodTypeFlow.java:112)
	at com.oracle.graal.pointsto.BigBang$1.run(BigBang.java:404)
	at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$execute$0(CompletionExecutor.java:173)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: java.lang.NoClassDefFoundError: okhttp3/logging/HttpLoggingInterceptor$Logger$DefaultImpls
	at parsing okhttp3.logging.HttpLoggingInterceptor$Logger.<clinit>(HttpLoggingInterceptor.kt:113)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2590)
	at com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:94)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3428)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3230)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1088)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:982)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:214)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:225)
	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:352)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.doParse(MethodTypeFlow.java:322)
	... 10 more
Caused by: java.lang.NoClassDefFoundError: okhttp3/logging/HttpLoggingInterceptor$Logger$DefaultImpls
	at java.base/java.lang.Class.getEnclosingMethod0(Native Method)
	at java.base/java.lang.Class.hasEnclosingMethodInfo(Class.java:1708)
	at java.base/java.lang.Class.isLocalOrAnonymousClass(Class.java:1704)
	at java.base/java.lang.Class.isLocalClass(Class.java:1656)
	at jdk.internal.vm.ci/jdk.vm.ci.hotspot.HotSpotJDKReflection.isLocalClass(HotSpotJDKReflection.java:94)
	at jdk.internal.vm.ci/jdk.vm.ci.hotspot.HotSpotResolvedObjectTypeImpl.isLocal(HotSpotResolvedObjectTypeImpl.java:938)
	at com.oracle.graal.pointsto.meta.AnalysisType.isLocal(AnalysisType.java:946)
	at com.oracle.svm.hosted.SVMHost.createHub(SVMHost.java:299)
	at com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:225)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:264)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:205)
	at com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupType(WrappedConstantPool.java:155)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupType(BytecodeParser.java:4274)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genNewInstance(BytecodeParser.java:4545)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5343)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3423)
	... 21 more
Error: Image build request failed with exit status 1
com.oracle.svm.driver.NativeImage$NativeImageError: Image build request failed with exit status 1
	at com.oracle.svm.driver.NativeImage.showError(NativeImage.java:1541)
	at com.oracle.svm.driver.NativeImage.build(NativeImage.java:1299)
	at com.oracle.svm.driver.NativeImage.performBuild(NativeImage.java:1260)
	at com.oracle.svm.driver.NativeImage.main(NativeImage.java:1219)
	at com.oracle.svm.driver.NativeImage$JDK9Plus.main(NativeImage.java:1722)
```

**Comment by mcraj017** [comment]
@zsiegel it seems to be related to this ticket 
https://github.com/square/okhttp/pull/5631
Check it out

**Comment by zsiegel** [comment]
I did some more digging. In this particular case for the logger it was fixed in a later commit which is below.

https://github.com/square/okhttp/commit/9724946320e8fdf94705b50dd85ad36ebd27cdc6#diff-c356c2713ee59a0de907dc0762afcd27

It is now fixed in 4.8.1 - I am now running into a new issue so I will close this and open a new issue for that.

Thanks!
